deploy_hpo.sh

```diff
@@ -24,24 +24,40 @@ echo
 echo "Using version: ${HPO_VERSION}"
```

Comment:
This print is misleading, include it only if user doesn't specify the hpo container image

Reply:
Done